### PR TITLE
Support command line

### PR DESCRIPTION
**Pull request by afc163**
```shell
$ npm install gh-pages -g
$ gh-pages -d dist -b master
```

### CODE REVIEW

**Comment by afc163** [comment]
ping~

File: bin/gh-pages [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+var path = require('path');
+ 
+program
+  .version(require('../package').version)
+  .option('-d, --dist <dist>',
+      'base directory for all source files')
+  .option('-s, --src <src>',
+      'pattern used to select which files should be published', '**/*')
+  .option('-b, --branch <branch>',
+      'name of the branch you\'ll be pushing to', 'gh-page')
```
Comment (tschaub) [inline bin/gh-pages]: Default should be `gh-page` instead of `gh-pages`, right?

Comment (tschaub) [inline bin/gh-pages]: Default should be `gh-pages` instead of `gh-page`, right?

**Comment by tschaub** [comment]
Thanks for the nice contribution @afc163!  I'll cut a release with this after a bit of testing later today.

**Comment by tschaub** [comment]
Thanks again for this addition @afc163.

**Pull request merged**